MatrixSDK/Utils/MXTools.m

```diff
@@ -827,8 +828,9 @@ + (void)convertVideoToMP4:(NSURL*)videoLocalURL
     // Convert video container to mp4
     // Use medium quality to save bandwidth
     AVURLAsset* videoAsset = [AVURLAsset URLAssetWithURL:videoLocalURL options:nil];
-    AVAssetExportSession *exportSession = [AVAssetExportSession exportSessionWithAsset:videoAsset presetName:AVAssetExportPresetMediumQuality];
+    AVAssetExportSession *exportSession = [AVAssetExportSession exportSessionWithAsset:videoAsset presetName:AVAssetExportPreset3840x2160];
```

Comment:
4K scares me :/

We should make it configurable with `MXSDKOptions` with a property like `videoConversionPresetName` and use HD as the default preset.

I do not think it worths to pass it as a parameter to `MXRoom.sendVideo()`(at least for the current behavior of Element).

Reply:
> 4K scares me :/

Agreed 😄

Added to `MXSDKOptions`.